cg/meta/report/field_validators.py

```diff
@@ -0,0 +1,56 @@
+"""Report field validation helper"""
+
+from cg.models.report.report import ReportModel
+
+
+def update_missing_data_dict(missing_data: dict, source: str, field: str, label: str = None):
+    """
+    Populates a specific dictionary with the missing/empty fields.
+    If the new entry is sample/application dependant field, the ID/TAG is also stored to uniquely identify it.
+    """
```

Comment:
I think this method might be easier to understand if the arguments were documented in the docstring:

```suggestion
    """
    Populates a specific dictionary with the missing/empty fields.
    If the new entry is sample/application dependant field, the ID/TAG is also stored to uniquely identify it.
    
    Args:
        param1 (int): The first parameter.
        param2 (str): The second parameter.

    Returns:
        bool: The return value. True for success, False otherwise.

    """
```

Reply:
Addressed in [65183d5](https://github.com/Clinical-Genomics/cg/pull/1438/commits/65183d5a915203911b484d7a38b157fa55095118)